src/scss/_action.scss

```diff
@@ -34,58 +34,44 @@
     justify-content: center;
 
     //content unique to this instance of c-action__link
-    background-color: $orange;
+    background-color: darken($green-brand, 2%);
```

Comment:
I love your choice here. Along with the header and new favicon, everything looks more cohesive.